src/VisualStudio/Core/Def/Implementation/FindReferences/AbstractAdditionalPropertyDefinition.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.VisualStudio.Shell.TableControl;
+
+namespace Microsoft.VisualStudio.LanguageServices.AdditionalProperty
```

Comment:
This should be moved to `Microsoft.VisualStudio.LanguageServices.FindUsages` namespace as this is specific to FAR columns.

Reply:
Please file a separate tracking issue for any refactoring/naming changes that you want to be done as a follow-up PR, in case you are unable to get to it in this PR.

Reply:
I opened this issue for followup: https://github.com/dotnet/roslyn/issues/38528